eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -3045,7 +3045,7 @@ public function testDeleteVersionWorksIfOnlyVersionIsDraft()
 
         // This call will fail with a "NotFound", because we allow to delete content if remaining version is draft.
         // Can normally only happen if there where always only a draft to begin with, simplifies UI edit API usage.
-        $this->contentService->loadContent($draft->id);
+        $this->contentService->loadContentInfo($draft->id);
```

Comment:
Can you explain why this is changed? I'd wager that both methods would return `NotFound` exception, as they attempt to load the same `Content` object, kind of.

Reply:
@Steveb-p I guess optimisation. There is no need to load and build fields if all we care is an exception (or nothing). This is +1 for me.

Reply:
Cool, just wanted to know what the reasoning was. I'm all for it then.